src/main/java/com/fabriccommunity/thehallow/TheHallowClient.java

```diff
@@ -1,35 +1,65 @@
-		BlockEntityRendererRegistry.INSTANCE.register(InfusionAltarBlockEntity.class, new InfusionAltarBlockEntityRenderer());
-		BlockEntityRendererRegistry.INSTANCE.register(HallowedTreasureChestBlockEntity.class, new HallowedTreasureChestBlockEntityRenderer());
+		BlockEntityRendererRegistry.INSTANCE.register(HallowedBlockEntities.TINY_PUMPKIN, TinyPumpkinRenderer::new);
+		BlockEntityRendererRegistry.INSTANCE.register(HallowedBlockEntities.INFUSION_PILLAR_BLOCK_ENTITY, InfusionPillarBlockEntityRenderer::new);
+		BlockEntityRendererRegistry.INSTANCE.register(HallowedBlockEntities.INFUSION_ALTAR_BLOCK_ENTITY, InfusionAltarBlockEntityRenderer::new);
+		BlockEntityRendererRegistry.INSTANCE.register(HallowedBlockEntities.HALLOWED_TREASURE_CHEST_BE, HallowedTreasureChestBlockEntityRenderer::new);
 
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.TAINTED_GLASS, RenderLayer.getTranslucent());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.TAINTED_GLASS_PANE, RenderLayer.getTranslucent());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.SOUL_GLASS, RenderLayer.getTranslucent());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.SOUL_GLASS_PANE, RenderLayer.getTranslucent());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.BREAD_CRUMBS, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.HALLOWED_GATE, RenderLayer.getTranslucent());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.HALLOWED_TREASURE_CHEST, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.INFUSION_ALTAR_BLOCK, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.INFUSION_PILLAR_BLOCK, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.RESTLESS_CACTUS, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.TINY_PUMPKIN, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.TOMBSTONE, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.WITCH_WATER_BUBBLE_COLUMN, RenderLayer.getTranslucent());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.BRAMBLES, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.DEADER_BUSH, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.GLOOMSHROOM, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.EERIE_GRASS, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.TALL_EERIE_GRASS, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.DEADWOOD_DOOR, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.DEADWOOD_TRAPDOOR, RenderLayer.getCutout());
+		BlockRenderLayerMap.INSTANCE.putBlock(HallowedBlocks.DEADWOOD_VINES, RenderLayer.getCutout());
+		
+		BlockRenderLayerMap.INSTANCE.putFluid(HallowedFluids.BLOOD, RenderLayer.getSolid());
```

Comment:
Do we actually want fluids to be on the solid render layer? I'd think we'd want them to be see-through a bit...

Reply:
This did stick out to me as odd, but this is how they are in 1.14 so I kept it.

Reply:
I think it's ok for now then. I'll take a look at some point.